editorialboard/models.py

```diff
@@ -150,13 +150,22 @@ class EditorialBoardMember(CommonControlField, Orderable):
     role = models.ForeignKey(
         "RoleModel", null=True, blank=True, related_name="+", on_delete=models.SET_NULL
     )
+    image = models.ForeignKey(
+        "wagtailimages.Image", 
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text=_("Upload a profile photo of the editorial board member."),
+    )
```

Comment:
@samuelveigarangel neste caso há um problema porque toda vez que o researcher for membro, terá que adicionar uma foto. Por outro lado se colocarmos a image no Researcher teremos muitos researcher, a maioria sem fotos, uma solução seria ter um BaseEditorialMember que teria researcher e image. Não sei qual seria a melhor opção.